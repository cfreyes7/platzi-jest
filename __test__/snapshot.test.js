import { getCharacter } from '../snapshot';
import rick from '../rick.json';

describe('Es hora de las intantaneas', () => {
    test('Snapshot', () => {
        expect(getCharacter(rick)).toMatchSnapshot();
    });
});