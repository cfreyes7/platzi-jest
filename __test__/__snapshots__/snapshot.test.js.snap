// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Es hora de las intantaneas Siempre fallara la instantanea 1`] = `
Object {
  "createAt": 2018-09-12T16:19:12.560Z,
  "id": 10,
}
`;

exports[`Es hora de las intantaneas Snapshot 1`] = `
Object {
  "gender": "Male",
  "id": 1,
  "name": "Rick Sanches",
  "species": "Human",
  "status": "Alive",
}
`;

exports[`Es hora de las intantaneas Tenemos una excepsion dentro del codigo 1`] = `
Object {
  "id": Any<Number>,
  "name": "Oscar Barajas",
}
`;
